Documentation/api-reference.md

```diff
@@ -138,14 +138,50 @@ sendEvent: function(xdmData as object, callback = _adb_default_callback as funct
 }
 ```
 
+> **Note**
+> `sendEvent` API allows passing custom identifiers to the edge network using custom Identity map . Create the Identity map using custom identifier namespace and configure the "primary" and "authenticatedState" for the individual identity as per the application requirements.
+
+> **Note**
+> Passing custom Identity map is optional. By Default SDK sends the IdentityMap with `ECID`. `ECID`` might be set as primary identifier on the edge network side if no other custom identifiers set as primary are sent.
+
+> **Warning**
+> Do not pass ECID in this IdentityMap. ECID will be attached by the SDK automatically.
+
+#### Sample Custom IdentityMap:
+
+```brightscript
+customIdentityMap = {
+    "CustomAdvertisingIdentifier" : [
+          {
+                "id" : "SampleAdIdentifier",
+                "authenticatedState": "ambiguous",
+                "primary": false
+          }
+    ],
+    "CustomIdentifierNamespace" : [
```

Comment:
suggest using a more realistic example, for example with RIDA and Email user@example.com